### DISCUSSION TITLE
Shutdown Tasks

### DISCUSSION THREAD

**Discussion by Technerder** [Minestom/Minestom#927] [Help] [answered]
Is there a way to register tasks that will be run when the `MinecraftServer.stopCleanly()` method is called?

**Comment by GoldenStack** [comment, marked as answer]
You can add a shutdown task via `SchedulerManager#buildShutdownTask(Runnable)`, but you can get there by doing something like `MinecraftServer.getSchedulerManager().buildShutdownTask(Runnable);`
